modules/Assay/Parameter.yaml

```diff
@@ -44,111 +17,153 @@ enums:
+        meaning: https://github.com/HumanCellAtlas/metadata-schema/blob/master/json_schema/type/process/sequencing/library_preparation_process.json
       single nucleus:
         description: Single nuclei
         meaning: https://github.com/HumanCellAtlas/metadata-schema/blob/master/json_schema/type/process/sequencing/library_preparation_process.json
+  ProteinExtractSourceEnum:
+    permissible_values:
+      cell lysate:
+        description: A material entity which is output of a cell lysis process
+        meaning: http://purl.obolibrary.org/obo/OBI_1000036
+      cytoplasm:
+        description: That portion of the cell contained within the plasma membrane but excluding the nucleus.
+        meaning: http://purl.obolibrary.org/obo/NCIT_C13226
       mitochondria:
         description: Mitochondria only
         meaning: https://github.com/HumanCellAtlas/metadata-schema/blob/master/json_schema/type/process/sequencing/library_preparation_process.json
+      nuclear extract:
+        description: Nuclear extracts contain proteins in nuclear compartment of the cell
+        meaning: http://purl.obolibrary.org/obo/NCIT_C19832
+  ReadPairOrientationEnum:
+    permissible_values:
+      fr-firststrand:
+        description: Equivalent to and should be migrated to 'first-stranded' under `libraryStrand`.
+      inward:
+        description: Reads face each other
+      matching:
+        description: Reads face the same direction
+      outward:
+        description: Reads face away from each other
+  ReadStrandOriginEnum:
+    permissible_values:
+      forward:
+        description: Read 1 (or unpaired read) comes from the forward strand and read 2 (if applicable) comes from the reverse strand
+        meaning: 'FALSE'
+      reverse:
+        description: read 1 (or unpaired read) comes from the reverse strand and read 2 (if applicable) comes from the forward strand
+        meaning: 'FALSE'
   RunTypeEnum:
     permissible_values:
-      singleEnd:
-        description: A library preparation that results in the creation of a library of 5' ends of DNA.
-        meaning: Sage Bionetworks
       pairedEnd:
         description: A library preparation that results in the creation of a library of the 5' and 3' ends of DNA or cDNA fragments using adaptors and endonucleases. The preparation may or may not include cloning process.
         meaning: http://purl.obolibrary.org/obo/OBI_0001852
-  ChannelEnum:
-    permissible_values:
-      Cy5:
-        description: Fluorescent dye used for the red channel on an array
-        meaning: http://purl.obolibrary.org/obo/FBbi_00000450
-      Cy3:
-        description: Fluorescent dye used for the green channel on an array
-        meaning: http://purl.obolibrary.org/obo/FBbi_00000449
-  GenePerturbationTechnologyEnum:
+      singleEnd:
+        description: A library preparation that results in the creation of a library of 5' ends of DNA.
+        meaning: Sage Bionetworks
+  StrandednessEnum:
```

Comment:
Do you think it is worth making this identical to HTAN's implementation of this concept? 
eg

term: Library Strand
values: Unstranded, FirstStranded, SecondStranded, NotApplicable

Reply:
Done